pkg/version/version.go

```diff
@@ -16,4 +16,4 @@
 
 package version
 
-const MonitoringAsCode = "1.5.0"
+const MonitoringAsCode = "config-poc"
```

Comment:
random note cause I started by scrolling to the bottom: this should change before merging this ;)